Terminal.Gui/Core/View.cs

```diff
@@ -181,15 +181,30 @@ internal enum Direction {
 		/// </summary>
 		public event Action<Key> HotKeyChanged;
 
+		Key hotKey = Key.Null;
```

Comment:
Why is the is `HotKey` stuff part of THIS PR? This PR is titled 'AutoSize is true by default on CheckBox`. I don't understand what hotkeys have to do with that. This PR changes 38 files!

Please try not to conflate multiple, separate/independent fixes into one PR. It will make it much harder to identify which PR did what in the future.

Reply:
I tried to create a PR with the `View` and `TextFormatter` and I get a lot of unit tests errors. To fix this it I also will end up with a big PR like this one and not worth to submit. So resolving this.